lib/render/Visualizer.cpp

```diff
@@ -584,7 +584,7 @@ int Visualizer:: _captureImage(std::string path)
         goto captureImageEnd;
     
     if (geoTiffOutput) {
-        string projString = _dataStatus->GetDataMgr(_dataStatus->GetDataMgrNames()[0])->GetMapProjection();
+        string projString = _dataStatus->GetDataMgr(_paramsMgr->GetDataMgrNames()[0])->GetMapProjection();
```

Comment:
At line 171, we guard against the case where there is no DataMgr, and _paramsMgr->GetDataMgrNames() returns an array of size 0.  Do we need to guard against that case here as well?

Reply:
Never hurts to add an assert. Done.